### ISSUE TITLE
Tab shows modified file when it is not

### ISSUE THREAD

**Issue by ivanddm** [ema2159/centaur-tabs#111]
Basically, the tabs are showing that a file is modified when it is actually not. I'm using Doom emacs, and this happens when I go into Normal mode (say with `jk`), the tab assumes that since I enter `jk` as characters that the buffer is modified when it is not. Changing tabs removes the 'modified' indicator from the tab.

This is my config:

```lisp
(use-package! centaur-tabs
  :config
  (setq centaur-tabs-style "rounded"
        centaur-tabs-set-icons t
        centaur-tabs-set-modified-marker t
        centaur-tabs-modified-marker "●"
        centaur-tabs-height 32
        centaur-tabs-set-bar 'over
        )
  (centaur-tabs-headline-match)
  (centaur-tabs-mode)
  ;; (defun centaur-tabs-buffer-groups ()
  ;;   (list "GROUP"))
  (defun centaur-tabs-hide-tab (x)
    "Disable tabs for certain buffer types"
    (let ((name (format "%s" x)))
      (or
       (window-dedicated-p (selected-window))
       (string-prefix-p "*epc" name)
       (string-prefix-p "*Messages" name)
       (string-prefix-p "*scratch" name)
       (string-prefix-p "*org-roam" name)
       (string-prefix-p "*doom" name)
       (string-prefix-p "*which-key*" name)
       (string-prefix-p "*Compile-Log*" name)
       (string-prefix-p "*tide-server" name)
       (string-prefix-p "*lsp" name)
       (and (string-prefix-p "magit" name)
            (not (file-name-extension name))))))
  :hook
  (org-mode . centaur-tabs-local-mode)
)
```

Thanks in advance.

**Comment by ema2159** [comment]
That's quite odd. Does the marker appears in all buffer types or in a specific buffer? After it appears, press `. in normal mode (go to the last modification in Evil/Vim), where does it land?

**Comment by ivanddm** [comment]
Yes it appears in all buffers, doesn't matter the file type. In normal mode `'.` takes me to the place where I just typed `jk` to exit insert mode.

These screenshots show that the the file is saved, not modified, as the directory color in the bottom is green in both cases. However, in the second one, the modified marker in the tab shows after exiting normal mode with `jk`.

<img width="407" alt="Screen Shot 2020-07-22 at 5 46 13 AM" src="https://user-images.githubusercontent.com/20104703/88128696-2c52f080-cbdf-11ea-98e4-8a39395def20.png">

<img width="485" alt="Screen Shot 2020-07-22 at 5 43 45 AM" src="https://user-images.githubusercontent.com/20104703/88128712-3248d180-cbdf-11ea-922c-bf7e23100d07.png">

**Comment by ema2159** [comment]
That's very uncommon. Is there anywhere where I can see your configuration?

**Comment by ivanddm** [comment]
> That's very uncommon. Is there anywhere where I can see your configuration?

Sure, here it is [my Doom Emacs config](https://github.com/ivanddm/.doom.d).
